### ISSUE TITLE
Image implementation for each items.

### ISSUE THREAD

**Issue by SushantGautam** [SushantGautam/NPbazzar#2]
This looks hard. .  but nothing is impossible. .  :)

**Comment by prasid444** [comment]
you want image for each list items?

**Comment by SushantGautam** [comment]
yeah! but dont waste your time collecting it for me, just kidding. :P I
think we should have url for each image stored in database. .  and then
render the image on app through internet/assets,. ,. ,. Lets see what
happens!

Sincerely,
_Sushant Gautam_
Srijan Galli, Kumarigal, Gaurighat-7, Chabahil, Kathmandu, Nepal
Cell: 977-9818342334
_http://www.sushant.info.np http://www.sushant.info.np_
Facebook Username: eSushant
Twitter: eSushant
